get_object/src/get_object.py

```diff
@@ -0,0 +1,24 @@
+import json
+import boto3
+
+dynamodb = boto3.resource('dynamodb')
+table = dynamodb.Table('kutter-table')
+
+def get_object_data(objectId):
+    response = table.get_item(
+        Key={
+            'objectId': objectId
+        }
+    )
+    return response['Item']
+
+def handler(event, context):
+    objectId = event ['queryStringParameters']['objectId']
+    item = get_object_data(objectId)
```

Comment:
when an object not exists you should return 404

Reply:
ok. will be done in issue: https://github.com/Chufle/kutter/issues/71